morse_code.rb

```diff
@@ -0,0 +1,25 @@
+def decode_char(morse_code)
+  morse_dict = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D',
+    '.' => 'E', '..-.' => 'F', '--.' => 'G', '....' => 'H',
+    '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+  morse_dict[morse_code].upcase
+end
+
```

Comment:
- [ ] You did a great job working on `decode_char` and `decode` methods. :+1: It seems the implementation for `decode_word` is missing. Kindly create the function to achieve the project requirements. 🤝️

**Requirements**
![image](https://github.com/Tafloninno/Morse_code_ruby/assets/17243325/312e3636-fb97-4720-8eb1-eac2d48c030e)

Reply:
- Well-resolved team 👍🏽